src/Test/Utilities/Portable/Compilation/CompilationExtensions.cs

```diff
@@ -321,13 +321,74 @@ void checkTimeout()
                             Assert.False(true, $"Duplicate explicit node for syntax ({operation.Syntax.RawKind}): {operation.Syntax.ToString()}");
                         }
                     }
-                    
+
                     visitor.Visit(operation);
                 }
+
+                stopWatch.Stop();
+                checkControlFlowGraph(root);
```

Comment:
> Is this actually taking long enough that we need to stop timing it? Seems like we should increase the timeout appropriately rather than stopping it.

I am comfortable with basing the timeout on the time it takes to verify original tree.

---
In reply to: [188726074](https://github.com/dotnet/roslyn/pull/26903#discussion_r188726074) [](ancestors = 188726074)